pkg/arvo/sys/vane/ames.hoon

```diff
@@ -528,6 +528,7 @@
           crypto-core=acru:ames
           =bug
       ==
+      ping=?  ::  is this a sponsor pinging channel?
```

Comment:
Putting this in between the "our" and "her" sections is a bit weird, I'd put it before or after `now`.